### DISCUSSION TITLE
I can not add the threshold pillar to <MINION_ID>.sls

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#2002] [Unsupported Versions] [answered]
Hello all,

I recently installed security onion 2.3.2 as a standalone system and a aim a newby in salt.

Now I am trying to add the threshold to the <MINION_ID> .sls. But when I give the command "sudo salt <MINION_ID> .sls state.apply idstools" I keep getting the following error in the master log file:

![afbeelding](https://user-images.githubusercontent.com/29328171/99701548-e296eb80-2a94-11eb-9bb4-4fd5fb9a83bb.png)

I have followed the following instruction of the documentation:
https://docs.securityonion.net/en/2.3/managing-alerts.html?highlight=threshold#threshold

This is my config of the idstools section in <MINION_ID> .sls

![Knipsel](https://user-images.githubusercontent.com/29328171/99702237-d5c6c780-2a95-11eb-9248-9cac1cdd0203.PNG)

Thx in advance 

Bart

**Comment by dougburks** [comment, marked as answer]
Have you double-checked your spacing? It looks like you might have some extra spaces compared to https://docs.securityonion.net/en/2.3/managing-alerts.html?highlight=threshold#disable-the-sid

**Reply by sleepingbel** [reply]
Hello Doug,

I copied the text of the threshold page and only changed the IP address.
I also checked manually whether there were extra spacings and this is not the case. Also i did not use the tab indentation 

I wil place the config file here, so you can check it also if you want

Thx for the response

**Reply by dougburks** [reply]
Perhaps I'm missing something, but looking at the config file you sent, the spacing in the idstools:sids:disabled section does not appear to match what's listed here:
https://docs.securityonion.net/en/2.3/managing-alerts.html?highlight=threshold#disable-the-sid

**Reply by sleepingbel** [reply]
This is correct, but on the server it is ok there is a space between the colon and ' . This part is accepted by the command. I tested it by deleting all the threshold lines.

But the threshold config gives always the same error in the master log file

to be complete:
- i did an iso install on an esxi 6.7
- i am using suricata

**Reply by dougburks** [reply]
Have you tried adding a colon after the SIDs?
```
    2028768:
```

```
    2000418:
```

**Reply by sleepingbel** [reply]
I tested this now and this was not the problem; This is the master error log 

![afbeelding](https://user-images.githubusercontent.com/29328171/99715606-64901000-2aa7-11eb-9d7b-e4363b0e1d0a.png)

**Reply by sleepingbel** [reply]
Thank you Doug fore all the help.

Found my mistake, it was a forgotten to place a spacing.

It works and I learned something about salt thanks

I Place my last config here, so it can help others

[SecOnion Threshold config1.txt](https://github.com/Security-Onion-Solutions/securityonion/files/5569427/SecOnion.Threshold.config1.txt)

Regards
Bart
